File: UI/LaundromatUI.Library/Models/CustomerModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LaundroDesktopUI.Library.Models
{
    public class CustomerModel
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public int Points { get; set; }

        public string Name => FirstName + " " + LastName;
    }
}

File: UI/laundromat-web-ui/src/app/models/customer.model.ts
export class Customer {
  id: number;
  firstName: string;
  lastName: string;
  email: string;
  phone: string;
  address: string;
  points: number;

  constructor(id:number = 0, firstName:string = "Guest", lastName:string = "",
  email:string = "", phone:string = "none", address:string = "", points:number = 0){
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.email = email;
    this.phone = phone;
    this.address = address;
    this.points = points;
  }

  getName(): string {
    return this.firstName + " " + this.lastName;
  }

}
